csm/core/controllers/view.py

```diff
@@ -180,6 +182,17 @@ def get_extension(filename):
         parse_results = {}
 
         reader = await request.multipart()
+        
+        try:
+            original_mask = os.umask(0o007)
+            Log.debug(f"original mask: {original_mask}")
+            FSUtils.create_dir(file_cache.cache_dir)
+        except Exception as e:
+            Log.debug(f"Can not create directory {e}")
```

Comment:
Should not continue if creation of cache_dir fails.

Reply:
Done.

Reply:
Done.